SOLIDITY_STYLE_GUIDE.md

```diff
@@ -1378,6 +1379,48 @@ function getStorageRoot(uint256 _blockHeight)
 }
 ```
 
+### Structs and Maps
+
+Document structs above the struct and struct fields above the respective field.
```

Comment:
Document structs above the struct declaration.
Document struct fields above the respective fields.